snowexsql/tables/doi.py

```diff
@@ -0,0 +1,28 @@
+from sqlalchemy import Column, String, Integer, ForeignKey, Date
+from sqlalchemy.orm import relationship, declared_attr
+
+from .base import Base
+
+
+class DOI(Base):
+    """
+    Table stores DOI values for each unique publication
+    """
+    __tablename__ = 'dois'
+
+    doi = Column(String())
+
+
+class HasOneDOI:
```

Comment:
Minor:
Since you name all the other ones "Has..", I think we should to the same here. I like your choice better as it is more concise.